pkg/defaults/mesh/mesh.go

```diff
@@ -61,10 +61,12 @@ func EnsureDefaultMeshResources(
 	}
 
 	defaultResourceBuilders := map[string]func() model.Resource{
-		"mesh-gateways-timeout-all": defaulMeshGatewaysTimeoutResource,
-		"mesh-timeout-all":          defaultMeshTimeoutResource,
-		"mesh-circuit-breaker-all":  defaultMeshCircuitBreakerResource,
-		"mesh-retry-all":            defaultMeshRetryResource,
+		"mesh-gateways-timeout-all-outbounds": defaultOutboundMeshGatewaysTimeoutResource,
```

Comment:
Looking at this code, I am afraid that this is not enough as we will have duplicate timeouts, if old defaults are present. 

Also, simply removing old defaults feels shady, what if someone updated defaults but left the default name?

Reply:
One more issue. What about zones running older versions? These policies won't work on zones

Reply:
@lukidzi can we protect ourselves from this? some flag on kds or something like this?

Reply:
It might be possible using a KDS flag, but we would need to translate it to a compatible format at the KDS sync level. That’s a bit tricky, and the global control plane would still display the new version. Since policies are created on the global control plane and it doesn't have awareness of the zone version, it's not possible to determine how to generate the default policy correctly.

Reply:
so should we then wait until we remove support for from entirely?